### PR TITLE
chore: removed the currency branch if no new commits

### PR DESCRIPTION
**Pull request by aryamohanan**
Currently, the currency-bot branch is created even when there are no version updates. This results in automatic updates failing because the branch already exists.

### CODE REVIEW

**Comment by kirrg001** [comment]
That is so cool

File: bin/currency/update-currencies.js [hunk 1]
```diff
@@ -124,11 +142,14 @@ currencies.forEach(currency => {
 
     if (!SKIP_PUSH) {
       execSync(`git push origin ${branchName} --no-verify`, { cwd });
-      execSync(
-        // eslint-disable-next-line max-len
-        `gh pr create --base main --head ${branchName} --title "[Currency Bot] Bumped ${currency.name} from ${installedVersion} to ${latestVersion}" --body "Tada!"`,
-        { cwd }
-      );
+
+      if (removeBranchIfNoCommits(branchName)) {
```
Comment (kirrg001) [inline bin/currency/update-currencies.js:146]: Suggestion: IMO there is no need to push the remote branch if there are no commits.

How about?

> if (hasCommitCount())
>   execSync(`git push origin ${branchName} --no-verify`, { cwd });
>   ...
> else
>   delete local branch

Comment (aryamohanan) [reply bin/currency/update-currencies.js]: Good suggestion ❤️

**Review by kirrg001 — APPROVED** [review]
LGTM
I did not test the `hasCommits` function. But if you have tested it, then it should be fine 👍

**Comment by aryamohanan** [comment]
Locally tested the changes.

**Pull request merged**